index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
Naming in general.

index.html is a standard name for the landing page.
In the case of a calculator I would have named it calculator.html / calculator.js